DownloadFiles.md

```diff
@@ -0,0 +1,79 @@
+- Images are returned at 100% scale. This may result in large images being returned, particularly for documents that were scanned at a high DPI (dots per inch).
+
+## Request format
+### Endpoint
+`POST /api/download`
+
+### HTTP header
+`Authorization: bearer {APIToken}`
+
+where `{APIToken}` is the user's UIX session token. For more information, see `apiAuthToken` defined in [Ringtail.Context](API.md#context).
+
+### Parameters
+
+Depending on how the UIX identifies the file to download, the required parameters vary.
+
+**To download a file based on the file ID:**
+
+| Parameter | Description | Type | Required? | Notes |
+| --- | --- | --- | --- | --- |
+| caseId | Case ID. Identifies the case in the portal. | int | Required |  |
+| fileId | File ID. Identifies the file in the case. | string | Required |  |
+
+For example: http://ringtail.example.com/Ringtail-Svc-Portal/api/download?caseId=6078&fileId=29384
+
+**To download a content file based on the content file type ranking:**
+
+| Parameter | Description | Type | Required? | Notes |
+| --- | --- | --- | --- | --- |
+| caseId | Case ID. Identifies the case in the portal. | int | Required |  |
+| mainId | Document Main ID. Identifies the document in the case. | int | Required |  |
+| contentFileRank | Rank of the content file displayed in the View pane. | int | Required | The highest ranked content file has a `contentFileRank` of 1. |
+
+For example: http://ringtail.example.com/Ringtail-Svc-Portal/api/download?caseId=3024&mainId=39456&contentFileRank=1
+
+**To download a page file based on the page file number:**
+
+| Parameter | Description | Type | Required? | Notes |
+| --- | --- | --- | --- | --- |
+| caseId | Case ID. Identifies the case in the portal. | int | Required |  |
+| mainId | Document Main ID. Identifies the document in the case. | int | Required |  |
+| pageFileNumber | Sequential order of the file among the document's page files. | int | Required |  |
+
+For example: http://ringtail.example.com/Ringtail-Svc-Portal/api/download?caseId=201&mainId=99810&pageFileNumber=2
+
+**To download a page file as a .jpg or .png image:**
+
+| Parameter | Description | Type | Required? | Notes |
+| --- | --- | --- | --- | --- |
+| caseId | Case ID. Identifies the case in the portal. | int | Required |  |
+| mainId | Document Main ID. Identifies the document in the case. | int | Required |  |
+| pageId | Page ID. Identifies the page file in the case. | int | Required |  |
+| pageNum | The number of the page that you want to return, for a page file that has multiple pages. | int | Required |  |
+| imageFormat | The file type that you want to return the image as. Specify one of `jpg` or `png`. | string | Optional | Default is jpg. |
+
+For example: http://ringtail.example.com/Ringtail-Svc-Portal/api/download?caseId=201&mainId=29384&pageId=1&pageNum=4&pageId=1&imageFormat=png
```

Comment:
`pageId` will be larger numbers, like 23901. Using an example of 1 is more misleading IMO.